src/pages/docs/providers/provider-uninstall.md

```diff
@@ -0,0 +1,97 @@
+---
+title: How to uninstall Golem Provider?
+description: Step-by-step guide to removing Yagna components, configs, and data for Providers.
+type: Instructions
+---
+
+# Golem Provider uninstallation guide
+
+{% alert level="warning" %}
+Before removing Yagna, please make sure you have backed up your private key. You can find the instructions [here](/docs/providers/wallet/backup).
+{% /alert %}
+
+## Quick path
+
+If you installed the provider using our quick-installer, execute the following commands to remove all associated components:
+
+```bash
+rm ~/.local/bin/yagna
+rm ~/.local/bin/golemsp
+rm ~/.local/bin/ya-provider
+rm -rf ~/.local/share/yagna
+rm -rf ~/.local/share/ya-installer
+rm -rf ~/.local/share/ya-provider
+rm -rf ~/.local/lib/yagna
+```
+
+## Detailed instructions for custom installations
+
+### Removing binaries
+
+Yagna binaries are typically located at: `~/.local/bin/`. Run the following command to remove all of the components.
+
+```bash
+rm ~/.local/bin/yagna && rm ~/.local/bin/golemsp && rm ~/.local/bin/ya-provider
+```
+
+### Removing Yagna binaries
+
+Yagna daemon configuration and persistent files are stored in the location defined by the `YAGNA_DATADIR` environment variable. If it was not set, then the default location for Yagna data is: `~/.local/share/yagna`.
+
+To verify if you use a custom data location, execute the following command:
+
+```bash
+echo $YAGNA_DATADIR
+```
+
+To remove the data in the default location, execute the command:
+
+```bash
+rm -r ~/.local/share/yagna
+```
+
+## Removing Yagna installer files
+
+To remove the data in the default location, execute the command:
```

Comment:
To me it's not clear what these files cover. I don't know whats in the directory and what its used for

Reply:
binaries are there